starter-code/index.html

```diff
@@ -10,15 +10,19 @@
     <link rel="stylesheet" href="styles/layout.css">
     <link rel="stylesheet" href="styles/modules.css">
     <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+    <script id="blog-articles" type="text/x-handlebars-template">
+      <article>
+        <header>
+          <h1>{{title}}</h1>
+          <div class="byline">
+              By <address><a href="{{authorUrl}}">{{author}}</a></address>
+              {{{publishStatus}}}
+          </div>
+        </header>
+        {{{body}}}
```

Comment:
This works. It displays the entire article. But now `setTeasers()` in `articleView.js` doesn't work.
I won't take points off this, but would love to see how you fix this issue :)